docs/en/tutorials/customize_rpn.md

```diff
@@ -0,0 +1,174 @@
+
+```python
+# training with 8 GPUS
+bash tools/dist_train.sh configs/faster_rcnn/faster-rcnn_r50_fpn_fcos-rpn_1x_coco.py \
+    8 \
+    --work-dir ./work_dirs/faster-rcnn_r50_fpn_fcos-rpn_1x_coco
+```
+
+## How to evaluate the quality of proposals
+
+The quality of proposals is of great importance to the performance of detector, therefore, we also provide a way to evaluate it. Same as above, create a new config file named `configs/rpn/fcos-rpn_r50_fpn_1x_coco.py`, and replace with setting of `rpn_head` with the setting of `bbox_head` in `configs/fcos/fcos_r50-caffe_fpn_gn-head_1x_coco.py`.
+
+```python
+_base_ = [
+    '../_base_/models/rpn_r50_fpn.py', '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+
+val_evaluator = dict(metric='proposal_fast')
+test_evaluator = val_evaluator
+
+model = dict(
+    # copied from configs/fcos/fcos_r50-caffe_fpn_gn-head_1x_coco.py
+    neck=dict(
+        start_level=1,
+        add_extra_convs='on_output',  # use P5
+        relu_before_extra_convs=True),
+    rpn_head=dict(
+        _delete_=True,  # ignore the unused old settings
+        type='FCOSHead',
+        num_classes=1,  # num_classes = 1 for rpn
+        in_channels=256,
+        stacked_convs=4,
+        feat_channels=256,
+        strides=[8, 16, 32, 64, 128],
+        loss_cls=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=1.0),
+        loss_bbox=dict(type='IoULoss', loss_weight=1.0),
+        loss_centerness=dict(
+            type='CrossEntropyLoss', use_sigmoid=True, loss_weight=1.0)))
+```
+
+Suppose we have the checkpoint `./work_dirs/faster-rcnn_r50_fpn_fcos-rpn_1x_coco/epoch_12.pth` after training, then we can evaluate the quality of proposals with the following command.
+
+```python
+# testing with 8 GPUs
+bash tools/dist_test.sh \
+    configs/rpn/fcos-rpn_r50_fpn_1x_coco.py \
+    ./work_dirs/faster-rcnn_r50_fpn_fcos-rpn_1x_coco/epoch_12.pth \
+    8
+```
+
+## How to train the customized Faster R-CNN with pre-trained FCOS
+
+Pre-training not only speeds up convergence of training, but also improves the performance of the detector. Therefore, here we give an example to illustrate how to do that. Suppose we want to use `FCOSHead` as an rpn head in Faster R-CNN and train with the pre-trained [`fcos_r50-caffe_fpn_gn-head_1x_coco`](https://download.openmmlab.com/mmdetection/v2.0/fcos/fcos_r50_caffe_fpn_gn-head_1x_coco/fcos_r50_caffe_fpn_gn-head_1x_coco-821213aa.pth). The content of config file named `configs/faster_rcnn/faster-rcnn_r50-caffe_fpn_fcos-rpn_1x_coco.py` is as follow. Note that `fcos_r50-caffe_fpn_gn-head_1x_coco` uses a caffe version of ResNet50, the pixel mean and std in `data_preprocessor` need be updated.
```

Comment:
Have you tried this before? maybe the key is not match, because in FCOS, keys of the head may name `detector.bbox_head.xxx` while in two-stage detector, it may name `detector.rpn.xxx`

Reply:
Not really sure, it is suggested to have a check